backend/src/auth/two-factor-auth/two-factor-auth.controller.ts

```diff
@@ -0,0 +1,89 @@
+import {
+	Body,
+	ClassSerializerInterceptor,
+	Controller,
+	Get,
+	HttpCode,
+	Injectable,
+	Post,
+	Req,
+	Res,
+	UnauthorizedException,
+	UseGuards,
+	UseInterceptors,
+} from '@nestjs/common';
+import { Request } from 'express';
+import { UserService } from 'src/user/user.service';
+import { RequestWithUser } from '../interface/requestWithUser.interface';
+import { TwoFactorAuthService } from './two-factor-auth.service';
+import { AuthenticatedGuard } from '../oauth/oauth.guard';
+import { TwoFactorAuthCodeDto } from '../dto/twoFactorAuthCode.dto';
+
+@Controller('two-factor-auth')
+@UseInterceptors(ClassSerializerInterceptor)
+// @Injectable()
+export class TwoFactorAuthController {
+	constructor(
+		private readonly twoFactorAuthenticationService: TwoFactorAuthService,
+		private readonly usersService: UserService,
+	) {}
+
+	// @Post('generate')
+	@Get('generate')
+	@UseGuards(AuthenticatedGuard)
+	//which guard should I use. if not JwtAuthenticationGuard
+	async register(@Res() response: Response, @Req() request: RequestWithUser) {
```

Comment:
I'm unable to start backend because of the error
     ```
                            ^
ReferenceError: Response is not defined
    at Object.<anonymous> (/ft_trans/backend/src/auth/two-factor-auth/two-factor-auth.controller.ts:35:34)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/olga/Study/ft_trans/backend/src/auth/auth.module.ts:10:1)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)


```